backend/app/routes/broadcast/getBroadcasts.js

```diff
@@ -3,32 +3,37 @@ const Broadcast = require('../../models/Broadcast');
 const { ErrorHandler } = require('../../../helpers/error');
 const constants = require('../../../constants');
 
+const limit = constants.PAGINATION_LIMIT.GET_BROADCASTS;
+
+const getBroadcastsAggregate = (startIndex, match) => {
+  const pipeline = [
+    { $match: match },
+    {
+      $project: {
+        _id: 1,
+        title: 1,
+        // imageUrl: 1,
```

Comment:
this should be present

Reply:
sorry about it, I forgot to uncomment it.